apps/researcher/src/components/add-object-list-form/actions.ts

```diff
@@ -0,0 +1,19 @@
+'use server';
+
+import {getCommunityById} from '@/lib/community';
+import {objectList} from '@colonial-collections/database';
+import {revalidatePath} from 'next/cache';
+
+interface ListItem {
+  communityId: string;
+  createdBy: string;
+  name: string;
+  description?: string;
+}
+export async function addList(listItem: ListItem) {
+  const response = await objectList.create(listItem);
+  const community = await getCommunityById(listItem.communityId);
+  revalidatePath(`/[locale]/communities/${community.slug}`, 'page');
+
+  return JSON.stringify(response);
```

Comment:
1. To be sure: do we use this response in other parts of the code?
2. Out of interest: is the `JSON.stringify()` required (i.e. Next doesn't do this for us)?

Reply:
I will first answer 2: This is required, without the `JSON.stringify()`, I get the warning 'Only plain objects can be passed to Client Components from Server Components'. 

However, we do not use the response. So I can also leave this line out.